nx/lib/nx.ex

```diff
@@ -4625,6 +4625,13 @@ defmodule Nx do
   For vectors with the same name to be compatible, they need to either
   have the same size or one must be of size 1.
 
+  ## Options
+
+    * `:make_broadcastable` - boolean that indicates whether the inner
+      shapes should also be modified. If `true`, 1-sized leading dimensions
```

Comment:
It seems we changed the default to `true`? Also, if the default is `true`, perhaps an option `vectors_only: true` would be better?

Reply:
Yes we did, as it is the most common use case and cleans up a lot of the expression graphs.

Reply:
Maybe "keep_inner_shape" instead? vectors only might convey that only tensors that are already vectorized would be changed